src/store/reducers/stateCache/stateCache.ts

```diff
@@ -12,7 +12,7 @@ import { LoginAttempts } from "../../../core/agent/services/auth.types";
 
 const initialState: StateCacheProps = {
   initialized: false,
-  isOnline: true,
+  isOnline: false,
```

Comment:
we are offline until we connect successfully!

Reply:
Set default value for `isOnline` is false maybe make all UT of all components are using useOnlineStatusEffect fail.